tools/proteore_topgo/topGO.xml

```diff
@@ -0,0 +1,268 @@
+        <param name="filetype" type="select" label="Enter your background IDs (Ensembl gene IDs)" help="(e.g : ENSG00000139618)">
+          <option value="file" selected="true">Input file containing your background IDs</option>
+          <option value="copy_paste">Copy/paste your background IDs</option>
+        </param>
+        <when value="copy_paste">
+          <param name="genelist" type="text" label="Copy/paste your background IDs" help="IDs must be separated by spaces into the form field, for example: ENSG00000139618 ENSG00000007350">
+            <sanitizer>
+            <valid initial="string.printable">
+                <remove value="&apos;"/>
+            </valid>
+            <mapping initial="none">
+                <add source="&apos;" target="__sq__"/>
+            </mapping>
+            </sanitizer>
+          </param>
+        </when>
+        <when value="file">
+          <param name="genelist" type="data" format="txt,tabular" label="Select file that contains your background IDs list" help=""/>
+          <param name="header" type="boolean" checked="true" truevalue="true" falsevalue="false" label="Does file contain header ?" />
+          <param name="column" type="text" label="Column number of IDs" value="c1" help="For example, fill in 'c1' if it is the first column, 'c2' if it is the second column and so on">
+            <validator type="regex" message="Please enter a column number, for example: 'c1' for the first column">[c]{0,1}[0-9]+</validator>
+          </param>
+        </when>
+      </conditional>
+    </when>
+    <when value="false"/>
+  </conditional>
+    <param name="geneuniverse" type="select" label="Species (human, mouse, rat)">
+      <!--option value="org.At.tair.db" >Arabidopsis</option-->
+      <!--option value="org.Ce.eg.db" >Worm (C. elegans)</option-->
+      <!--option value="org.Dm.eg.db" >Fly (D. melanogaster)</option-->
+      <option value="org.Hs.eg.db" selected="true">Human (Homo sapiens)</option>
+      <option value="org.Mm.eg.db" >Mouse (Mouse musculus)</option>
+      <option value="org.Rn.eg.db" >Rat (Rattus norvegicus)</option>
+      <!--option value="org.Sc.sgd.db" >Yeast (S. cerevisiae)</option-->
+    </param>
+    <param name="ontocat" type="select" label="GO terms category">
+      <option value="BP" >Biological Process</option>
+      <option value="CC" >Cellular Component</option>
+      <option value="MF" >Molecular Function</option>
+    </param>
+    <param name="option" type="select" label="Select GO scoring method (read user doc section)">
+      <option value="classic" >Classic Fisher test</option>
+      <option value="elim" selected="true">Elim</option>
+      <option value="weight01" >Weight01</option>
+      <option value="parentchild" >ParentChild</option>
+    </param>
+    <param name="threshold" type="text" label="p-value threshold (e.g : 1e-3)" value="1e-3"/>
+	  <param name="correction" label="Multiple testing procedure (p-value adjustment)" type="select">
```

Comment:
indentation ... I recommend using https://github.com/galaxyproject/galaxy-language-server it will format your files nicely and will point you to a few limiting issues.